src/app/modules/side-menu/components/filter-menu/filter-menu.component.ts

```diff
@@ -124,6 +125,29 @@ export class FilterMenuComponent extends NgDestoryBase implements OnInit {
   private http = inject(HttpClient);
   private logger = inject(LoggingService);
   private searchCriteriaModelService = inject(SearchCriteriaModelService);
+  observationGroups = toSignal(this.searchCriteriaModelService.getObservationGroups$());
```

Comment:
Tror det er bedre å kalle denne og metoden i searchCriteriaModelService for observationTypeGroups, for å unngå at det blandes med gruppene en observasjon kan deles med.